README.md

```diff
@@ -52,10 +52,11 @@ Since v0.15.0, httpin also supports creating an HTTP request (`http.Request`) fr
 
 ```go
 type ListUsersInput struct {
-	Token    string `in:"query=access_token;header=x-access-token"`
-	Page     int    `in:"query=page;default=1"`
-	PerPage  int    `in:"query=per_page;default=20"`
-	IsMember bool   `in:"query=is_member"`
+	Token    string  `in:"query=access_token;header=x-access-token"`
+	Page     int     `in:"query=page;default=1"`
+	PerPage  int     `in:"query=per_page;default=20"`
+	IsMember bool    `in:"query=is_member"`
+	Search   *string `in:"query=search,omitempty"`
```

Comment:
The `query` directive is designed in the signautre as `query(name1, name2, ...)`, here the `omitempty` will be read as the second name used in the query parameters. I'm thinking:

1. if we should define a separate `omitempty` directive for this, i.e. `in:"query=search;header=x-api-keyword;omitempty"`?
2. shall we just use `patch.Field[T]` as the field type, since it was designed for indicating **absent values**, i.e. `Search patch.Field[string]`, when `Search.Valid == false`, it will be omitted from the final resource.

Reply:
Using `patch.Field[T]` for the field type would not be ideal for us as this would complicate using our library for consumers, if this can be handled purely by the stuct tags then this would be ideal.

I have updated it to use a new `omitempty` directive now. I do agree it is better.

I'm not sure on the implementation - as the new directive doesn't take any action by itself - but is used inside the existing directives - maybe this is ok, or maybe there is a better way?

Reply:
I totally agree. I hope `patch.Field[T]` should be used only in rare cases.